x/ibc/core/04-channel/types/codec.go

```diff
@@ -14,25 +14,16 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterInterface(
 		"ibc.core.channel.v1.ChannelI",
 		(*exported.ChannelI)(nil),
+		&Channel{},
 	)
 	registry.RegisterInterface(
 		"ibc.core.channel.v1.CounterpartyChannelI",
 		(*exported.CounterpartyChannelI)(nil),
+		&Counterparty{},
 	)
 	registry.RegisterInterface(
 		"ibc.core.channel.v1.PacketI",
 		(*exported.PacketI)(nil),
-	)
-	registry.RegisterImplementations(
```

Comment:
Is this no longer needed @colin-axner ?

Reply:
Duplicate registration. `RegisterInterface` will register implementations as well